app/feature/feature-changeaddress/src/main/kotlin/com/hedvig/android/feature/changeaddress/destination/ChangeAddressOfferDestination.kt

```diff
@@ -433,12 +361,15 @@ private fun PreviewChangeAddressOfferScreen() {
           moveIntentId = MoveIntentId(""),
           quotes = List(2, MoveQuote::PreviewData),
           movingDate = ValidatedInput(Clock.System.now().toLocalDateTime(TimeZone.UTC).date),
+          extraBuildingTypes = emptyList(),
         ),
         {},
         {},
         {},
         {},
         {},
+        {},
+        imageLoader = PreviewImageLoader(LocalContext.current),
```

Comment:
Can use just `rememberPreviewImageLoader()` to not have to call the LocalContext
```suggestion
        imageLoader = rememberPreviewImageLoader(),
```